spec/controllers/anime_controller_spec.rb

```diff
@@ -1,3 +1,47 @@
+# rubocop:disable Metrics/LineLength
+# == Schema Information
+#
+# Table name: anime
+#
+#  id                        :integer          not null, primary key
+#  abbreviated_titles        :string           is an Array
+#  age_rating                :integer          indexed
+#  age_rating_guide          :string(255)
+#  average_rating            :float            indexed
+#  canonical_title           :string           default("en_jp"), not null
+#  cover_image_content_type  :string(255)
+#  cover_image_file_name     :string(255)
+#  cover_image_file_size     :integer
+#  cover_image_top_offset    :integer          default(0), not null
+#  cover_image_updated_at    :datetime
+#  end_date                  :date
+#  episode_count             :integer
+#  episode_length            :integer
+#  poster_image_content_type :string(255)
+#  poster_image_file_name    :string(255)
+#  poster_image_file_size    :integer
+#  poster_image_updated_at   :datetime
+#  rating_frequencies        :hstore           default({}), not null
+#  show_type                 :integer
+#  slug                      :string(255)      indexed
+#  start_date                :date
+#  started_airing_date_known :boolean          default(TRUE), not null
+#  synopsis                  :text             default(""), not null
+#  titles                    :hstore           default({}), not null
+#  user_count                :integer          default(0), not null, indexed
+#  created_at                :datetime         not null
+#  updated_at                :datetime         not null
+#  youtube_video_id          :string(255)
+#
+# Indexes
+#
+#  index_anime_on_age_rating  (age_rating)
+#  index_anime_on_slug        (slug) UNIQUE
+#  index_anime_on_user_count  (user_count)
+#  index_anime_on_wilson_ci   (average_rating)
+#
+# rubocop:enable Metrics/LineLength
+
```

Comment:
Would you mind removing these comments in `spec/controllers` from your PR?  It's not your fault, I just can't get the `annotate` gem to stop putting them in these files until ctran/annotate_models#370 lands, so we're just stuck manually removing them whenever it tries to add them :/